doc/api/util.markdown

```diff
@@ -194,6 +194,21 @@ Returns `true` if the given "object" is a `RegExp`. `false` otherwise.
     util.isRegExp({})
       // false
 
+note: This function uses `Object.prototype.toString.call` and check the returned value is `[object RegExp]`. However, `Symbol.toStringTag` can change the returned value directly. `Symbol.toStringTag` is enabled under `--harmony-tostring`.
```

Comment:
"Note:" (capitalize) and please wrap lines at 80 columns.  I'd write 'enabled with' instead of 'enabled under'.